pyproject.toml

```diff
@@ -10,6 +10,10 @@ python = "^3.8"
 virtool-workflow = {git = "https://github.com/virtool/virtool-workflow", rev = "develop"}
 
 [tool.poetry.dev-dependencies]
+pytest-aiohttp = "^0.3.0"
+pytest-asyncio = "^0.15.1"
```

Comment:
Pretty sure only one of these should be used. The better one is `pytest-aiohttp` in my opinion.